glue/configure_includes/configure_linux_aarch64.mk

```diff
@@ -0,0 +1,61 @@
+###############################################################################
+# Copyright (c) 2018, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+include $(CONFIG_INCL_DIR)/configure_common.mk
+
+#	TODO:AARCH64 Disable fvtest ???
```

Comment:
What are the pros and cons here? I guess the issue is if we are cross-compiling we won't be able to execute fvtests on the build platform. Is there merit in building the fvtests (but not executing them, obviously) so that the build directory could then be mounted/copied to an aarch64 platform to run the tests? I guess other option is to take fvtest out of the cross compile spec and add a new spec for native compile which includes it.
Either way we should probably make a decision and implement whatever that implies for this PR before merging :)

Reply:
I'd say we should build them so that they could be migrated to an AArch64 platform to be natively executed. Thoughts?

Reply:
That sounds very reasonable to me.

Reply:
My thoughts too. That was the idea behind this. At least if we use this for jenkins